sources/packages/backend/apps/api/src/route-controllers/program-info-request/program-info-request.institutions.controller.ts

```diff
@@ -250,6 +251,7 @@ export class ProgramInfoRequestInstitutionsController extends BaseController {
           case APPLICATION_NOT_FOUND:
             throw new NotFoundException(error.message);
           case STUDY_DATE_OVERLAP_ERROR:
+          case EDUCATION_PROGRAM_IS_NOT_ACTIVE:
```

Comment:
Is there a requirement to show this error in the UI (snackbar) ?

as it is being in the group of errors which are throwing APIProcessError?